### PR TITLE
Display additional fields when listing DatabaseClusters

### LINKED ISSUE #15: Display additional fields by default when listing DatabaseClusters
Currently when listing `DatabaseCluster` resources, you see only the default columns:

```sh
$ kubectl get databasecluster
NAME                    AGE
sample-mysql-database   7d4h
```

It would be nice to display the database engine, DO database name, and database status as well.

This can be done with [kubebuilder annotations](https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns).

### PR DESCRIPTION
**Pull request by mo-rieger**
CLOSES #15

### CODE REVIEW

File: api/v1alpha1/databasecluster_types.go [hunk 1]
```diff
@@ -75,6 +75,9 @@ type DatabaseClusterStatus struct {
 //+kubebuilder:subresource:status
 
 // DatabaseCluster is the Schema for the databaseclusters API
+// +kubebuilder:printcolumn:name="Engine",type=string,JSONPath=`.spec.engine`
+// +kubebuilder:printcolumn:name="DO name",type=string,JSONPath=`.spec.name`
```
Comment (adamwg) [inline api/v1alpha1/databasecluster_types.go:79]: I think `DO name` might be confusing for users. I'd suggest maybe `Cluster name` instead.

Comment (mo-rieger) [reply api/v1alpha1/databasecluster_types.go]: Agree

File: config/crd/bases/databases.digitalocean.com_databaseclusters.yaml [hunk 2]
```diff
@@ -15,7 +15,17 @@ spec:
     singular: databasecluster
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .spec.engine
+      name: Engine
+      type: string
+    - jsonPath: .spec.name
+      name: DO name
```
Comment (adamwg) [inline config/crd/bases/databases.digitalocean.com_databaseclusters.yaml:23]: Need a `make manifests` to update this, I believe :-)

Comment (mo-rieger) [reply config/crd/bases/databases.digitalocean.com_databaseclusters.yaml]: Sorry, I forgot this. Updated now.

**Review by adamwg — APPROVED** [review]
lgtm

**Pull request merged**